eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -63,17 +76,26 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
         parameters:
           NuGetCacheKey: Test
+      - pwsh: | 
+          if ("${{ parameters.ProjectListOverrideFilePropertyName }}" -ne '') {
+            Write-Host "##vso[task.setvariable variable=ProjectListFileName]$(${{ parameters.ProjectListOverrideFilePropertyName }})"
+          }
+          else 
+          {
+            Write-Host "##vso[task.setvariable variable=ProjectListFileName]"
```

Comment:
Lets call this variable ProjectListOverrideFile to align with the property we are going to pass.

Reply:
DevOps doesn't like the idea of overwriting the var:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1761581&view=logs&jobId=0df9fa30-75ef-5c92-acf8-51db11a32c21&j=0ed1bd63-0020-5305-9305-7dd16ef5986a&t=dc98aaf6-d56e-50eb-bcad-92c9878fbc82

<img width="554" alt="image" src="https://user-images.githubusercontent.com/48036328/183140951-c888a189-701c-4170-a1d2-a73ab07ef0bf.png">

Reply:
This error is because the variable isn't set though?

Reply:
Sorry paste wrong screenshot. Not able find the pipeline. Will trigger another time.

Reply:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1762037&view=logs&jobId=6a7c8c86-4e1c-58e5-a505-ef9edcd38ccb&j=6a7c8c86-4e1c-58e5-a505-ef9edcd38ccb&t=162e7796-e100-5fe5-48d1-6a37d9d08543

<img width="1047" alt="image" src="https://user-images.githubusercontent.com/48036328/183157444-531e04b2-d15c-4007-92a8-b48b439de330.png">